Week1/homework/app.js

```diff
@@ -1,61 +1,145 @@
 'use strict';
-
 {
-	const gameTitles = [
-		'stardew_valley',
-		'overwatch',
-		'lineage',
-		'counter_strike',
-		'wow',
-		'house_of_the_dead'
-	];
-
-	let objGames = {
-		"stardew_valley":
+	let games = [
 		{
+			title: 'stardew_valley',
 			year: '2016',
 			company: 'ConcernedApe',
-			type: 'role play'
+			type: 'role play',
+			image: 'images/stardewvalley.jpg'
 		},
-		"overwatch":
 		{
+			title: 'overwatch',
 			year: '2016',
 			company: 'blizzard',
-			type: 'shooter'
+			type: 'shooter',
+			image: 'images/overwatch.png'
 		},
-		'lineage':
 		{
+			title: 'lineage',
 			year: '2003',
 			company: 'NCSOFT',
-			type: 'MMORPG'
+			type: 'MMORPG',
+			image: 'images/lineage.png'
 		},
-		'counter_strike':
 		{
+			title: 'counter_strike',
 			year: '2000',
 			company: 'namco',
-			type: 'shooter'
+			type: 'shooter',
+			image: 'images/counterstrike.jpg'
 		},
-		'wow': {
+		{
+			title: 'wow',
 			year: '2004',
 			company: "blizzard",
-			type: 'role play'
+			type: 'role play',
+			image: 'images/wow.jpg'
 		},
-		'house_of_the_dead': {
+		{
+			title: 'house_of_the_dead',
 			year: '1996',
 			company: 'sega',
-			type: 'light gun shooter'
+			type: 'light gun shooter',
+			image: 'images/hotd.jpg'
+		}
+	]
+
+	document.body.onload = listGames;
+
+	function listGames() {
+		let ul = document.createElement('ul')
+		for (const game of games) {
+			let li = document.createElement('li');
+			li.innerHTML = `<h2>${game.title}</h2>`
+			li.innerHTML += `Year: ${game.year}<br>`
+			li.innerHTML += `Type: ${game.type}<br>`
+			li.innerHTML += `Company: ${game.company}<br>`
+			let cover = document.createElement("img");
+			cover.width = 250
+			cover.src = game.image;
+			li.appendChild(cover)
+			ul.appendChild(li);
 		}
+		document.body.appendChild(ul);
```

Comment:
Nice clean code. Very understandable. However, is the spacing right? Feels _very_ indented.

Reply:
I guess you mean that here it starts like in the middle of the page? I had to google what indented means hahaha but on visualcode looks different with a lot less spaces in the beginning